client/handle_http.go

```diff
@@ -1698,6 +1700,7 @@ func downloadObject(transfer *transferFile) (transferResults TransferResults, er
 			break
 		}
 	}
+	transferResults.TransferStartTime = transferStartTime
```

Comment:
It looks like the `TransferStartTime` is the start time of the last transfer attempt (successful or not). Could you leave a comment here or around line 1645 where you defined this variable? I'm was a bit confused when I first saw its definition and where it gets used.

Reply:
Also, I was wondering if this is actually the value we want to record? Why not recording the time at the start of the first attempt?